sfs/time/source.py

```diff
@@ -79,13 +79,16 @@ def point(xs, signal, observation_time, grid, c=None):
     if c is None:
         c = default.c
     r = np.linalg.norm(grid - xs)
-    # evaluate g over grid
-    weights = 1 / (4 * np.pi * r)
+    # evaluate g over grid (`r` can get `0`, which is fine)
```

Comment:
What about

```python
# If r is +- 0, the sound pressure is +- infinity
```

BTW, if the comment "evaluate g over grid" is really necessary, it should go directly before the calculation, not before the `with` statement.

Reply:
Changed it accordingly, and removed "evaluate g over grid"